.github/policy.rego

```diff
@@ -27,6 +27,14 @@ deny_unsafe_checkout[msg] {
     msg := "Explicit checkout in a pull_request_target workflow is unsafe. See https://securitylab.github.com/resources/github-actions-preventing-pwn-requests for more information."
 }
 
+deny_unnecessary_github_token[msg] {
+    some job in input["jobs"]
+    some step in job["steps"]
+    startswith(step["uses"], "actions/github-script@")
```

Comment:
What is going to happen for the existing usage like https://github.com/mlflow/mlflow/blob/master/.github/workflows/rerun-workflow-run.yml#L38?

Reply:
@TomeHirata No behavior change. `github-token` would default to `github.token` which is an alias of `${{ secrets.GITHUB_TOKEN }}`.

Reply:
Sorry if my question is not clear, can I know if it does not cause any issue if we merge this change before changing the existing usecases like [this](https://github.com/mlflow/mlflow/blob/master/.github/workflows/rerun-workflow-run.yml#L38)?

Reply:
No, this PR will break nothing :)